<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>606</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800000</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Black Kicked Out</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="list_black_kickout">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>White Kicked Out</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListWidget" name="list_white_kickout">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>FEN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>60</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>60</number>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>4</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>5</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>6</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>7</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>8</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>a</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>b</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>c</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>d</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>e</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>f</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>g</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>h</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Log</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTextEdit" name="log">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>500000</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Import/Export</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Import</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1096</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHuman_vs_Human">
    <property name="title">
     <string>Gamemode</string>
    </property>
    <addaction name="actionHuman_vs_Human"/>
    <addaction name="actionHuman_vs_PC"/>
    <addaction name="actionPC_vs_PC"/>
   </widget>
   <addaction name="menuHuman_vs_Human"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHuman_vs_Human">
   <property name="text">
    <string>Human vs Human</string>
   </property>
  </action>
  <action name="actionHuman_vs_PC">
   <property name="text">
    <string>Human vs PC</string>
   </property>
  </action>
  <action name="actionPC_vs_PC">
   <property name="text">
    <string>PC vs PC</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
